recipes/folly/all/conandata.yml

```diff
@@ -6,51 +6,94 @@ sources:
     url: "https://github.com/facebook/folly/archive/v2020.08.10.00.tar.gz"
     sha256: "e81140d04a4e89e3f848e528466a9b3d3ae37d7eeb9e65467fca50d70918eef6"
   "2022.01.31.00":
-    url: "https://github.com/facebook/folly/archive/v2022.01.31.00.tar.gz"
-    sha256: "d764b9a7832d967bb7cfea4bcda15d650315aa4d559fde1da2a52b015cd88b9c"
+    url: "https://github.com/facebook/folly/releases/download/v2022.01.31.00/folly-v2022.01.31.00.tar.gz"
+    sha256: "7b8d5dd2eb51757858247af0ad27af2e3e93823f84033a628722b01e06cd68a9"
+  "2022.10.31.00":
+    url: "https://github.com/facebook/folly/releases/download/v2022.10.31.00/folly-v2022.10.31.00.tar.gz"
+    sha256: "d7749f78eee2a327c1fa6b4a290e4bcd7115cdd7f7ef59f9e043ed59e597ab30"
 patches:
   "2019.10.21.00":
     - patch_file: "patches/0001-find-packages.patch"
-      base_path: "source_subfolder"
+      patch_description: "fix folly-deps.cmake"
+      patch_type: "conan"
     - patch_file: "patches/0002-compiler-flags.patch"
-      base_path: "source_subfolder"
+      patch_description: "fix compiler flags"
+      patch_type: "conan"
     - patch_file: "patches/0003-boost-shared-ptr.patch"
-      base_path: "source_subfolder"
+      patch_description: "fix boost shared ptr"
+      patch_type: "conan"
     - patch_file: "patches/0004-disable-posix-names.patch"
-      base_path: "source_subfolder"
+      patch_description: "disable posix names"
+      patch_type: "conan"
     - patch_file: "patches/0005-include-atomic.patch"
-      base_path: "source_subfolder"
+      patch_description: "include atomic"
+      patch_type: "conan"
     - patch_file: "patches/0006-duplicate-hash.patch"
-      base_path: "source_subfolder"
+      patch_description: "duplicate hash"
+      patch_type: "conan"
     - patch_file: "patches/0007-allow-builtins.patch"
-      base_path: "source_subfolder"
+      patch_description: "allow builtins"
+      patch_type: "conan"
     - patch_file: "patches/0013-include-bit.patch"
-      base_path: "source_subfolder"
+      patch_description: "include bit"
+      patch_type: "conan"
     - patch_file: "patches/0020-include-ssizet.patch"
-      base_path: "source_subfolder"
+      patch_description: "include ssizet"
+      patch_type: "conan"
+    - patch_file: "patches/0023-fix-cmake-generator-expr.patch"
+      patch_description: "CMake: Handle generator expressions requiring a target for pkg-config #1433"
+      patch_type: "backport"
```

Comment:
I've seen that the linter flags this "backport" as incorrect. This patch refers to this https://github.com/facebook/folly/pull/1433 (I will add it as `patch_source` later). 

But my doubt is how it should be classified. As `"bugfix"` or `"portability"`? Any suggestion?

Reply:
Ughs... Conan :thinking_face: this is a consequence of the targets conan is injection